ui/framework/src/ui-framework/imodel-components/category-tree/CategoriesTree.tsx

```diff
@@ -111,29 +113,24 @@ export function CategoryTree(props: CategoryTreeProps) {
   }, []);
 
   return (
-    <div className="ui-fw-categories-tree">
+    <div className="ui-fw-categories-tree" style={{ width: props.width, height: props.height }}>
       <ControlledTree
         nodeLoader={filteredNodeLoader}

```

Comment:
What happens when we do not set explicit dimensions on this `div`?

Reply:
Cleaned up